packages/stripe_platform_interface/lib/src/models/errors.dart

```diff
@@ -30,6 +30,21 @@ T _dataFromJson<T>(Map<String, dynamic> input) => input['code'] as T;
 Map<String, dynamic> _dataToJson<T>(T input) => {'code': input};
 
 @freezed
+
+/// Exception retrieved from the Stripe platform.
+class StripePlatformException with _$StripePlatformException {
```

Comment:
I'd prefer StripeError - "Stripe platform" sounds somewhat misleading

Reply:
I wouldn't call it then `StripeError` because of the following:
- error is a problem that the user should not catch , most of the time something unforseen happens.
-  exception is a problem that the user might want to catch.

agree?